packages/preview/invoice-maker/1.0.0/lib.typ

```diff
@@ -0,0 +1,448 @@
+#let nbh = "‑"
+
+// Truncate a number to 2 decimal places
+// and add trailing zeros if necessary
+// E.g. 1.234 -> 1.23, 1.2 -> 1.20
+#let add_zeros = (num) => {
+    // Can't use trunc and fract due to rounding errors
+    let frags = str(num).split(".")
+    let (intp, decp) = if frags.len() == 2 { frags } else { (num, "00") }
+    str(intp) + "." + (str(decp) + "00").slice(0, 2)
+  }
+
+// From https://stackoverflow.com/a/57080936/1850340
+#let verify_iban = (country, iban) => {
+    let iban_regexes = (
+        DE: regex(
+          "^DE[a-zA-Z0-9]{2}\s?([0-9]{4}\s?){4}([0-9]{2})$"
+        ),
+        GB: regex(
+          "^GB[a-zA-Z0-9]{2}\s?([a-zA-Z]{4}\s?){1}([0-9]{4}\s?){3}([0-9]{2})$"
+        ),
+      )
+
+    if country == none or not country in iban_regexes {
+      true
+    }
+    else {
+      iban.find(iban_regexes.at(country)) != none
+    }
+}
+
+#let TODO = box(
+  inset: (x: 0.5em),
+  outset: (y: 0.2em),
+  radius: 0.2em,
+  fill: rgb(255,180,170),
+)[
+  #text(
+    font: "Arial",
+    size: 8pt,
+    weight: 600,
+    fill: rgb(100,68,64)
+  )[TODO]
+]
+
+#let horizontalrule = [
+  #v(8mm)
+  #line(
+    start: (20%,0%),
+    end: (80%,0%),
+    stroke: 0.8pt + gray,
+  )
+  #v(8mm)
+]
+
+#let signature_line = line(length: 5cm, stroke: 0.4pt)
+
+#let endnote(num, contents) = [
+  #stack(dir: ltr, spacing: 3pt, super[#num], contents)
+]
+
+#let languages = (
```

Comment:
Is there some way to customize those as a user or add a new language? I think that would be really useful.